go.mod

```diff
@@ -36,13 +41,4 @@ require (
 	gopkg.in/ini.v1 v1.67.0 // indirect
 )
 
-require (
-	github.com/golang-jwt/jwt/v4 v4.5.0
-	github.com/lufia/plan9stats v0.0.0-20230110061619-bbe2e5e100de // indirect
-	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
-	github.com/power-devops/perfstat v0.0.0-20221212215047-62379fc7944b // indirect
-	github.com/prometheus/client_model v0.4.0
-	github.com/prometheus/prom2json v1.3.3
-)
-
 replace github.com/minio/minio-go/v7 => github.com/poornas/minio-go/v7 v7.0.0-20230816201254-210e170bdc4c
```

Comment:
we need to remove this as well - is this added by you @poornas ?

Reply:
yes it seems so, not sure if it was intentional and temporary.

Reply:
minio-go is released so remove this and use the latest release @cesnietor

Reply:
done.